### PR TITLE
added pip and pypi (English)

### PR DESCRIPTION
**Pull request by callumrollo**
## Author: 

- Callum Rollo

## Language: 

- English

## Terms defined:

- PyPI
- pip

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -4481,6 +4481,17 @@
...
 
+- slug: pip
+  ref:
+    - python
+  en:
+    term: "Pip Install Packages"
+    acronym: "PIP"
+    def: >
+      The standard [package manager](#package_manager) for [Python](#python). pip Enables the download and installation
+      of Python packages not included in the standard library.
```
Comment (baileythegreen) [inline glossary.yml:4492]:
```suggestion
      The standard [package manager](#package_manager) for [Python](#python). pip enables
      the download and installation of Python packages not included in the standard library.
```


We try to keep lines from getting too wide—no set character limit, but use some longer entries in the file to see where to put line breaks; but don't split words.

Comment (baileythegreen) [reply glossary.yml:4492]: Also, `enables` didn't need to be capitalised.

File: glossary.yml [hunk 2]
```diff
@@ -4740,6 +4751,17 @@
...
 
+- slug: pypi
+  ref:
+    - python
+  en:
+    term: "Python Package Index"
+    acronym: "PyPI"
+    def: >
+      The official third-party software repository for [Python](#python). Anyone can upload a [package](#package)
+      to PyPI. PyPI packages may install via executed scripts or pre-compiled system-specific wheels.
```
Comment (baileythegreen) [inline glossary.yml:4762]:
```suggestion
      The official third-party software repository for [Python](#python). Anyone can upload
      a [package](#package) to PyPI. PyPI packages may install via executed scripts or
      pre-compiled, system-specific wheels.
```

Line breaks again.

File: glossary.yml [hunk 3]
```diff
@@ -4740,6 +4751,17 @@
       easily produce gaps and collisions.
 
 
+- slug: pypi
+  ref:
+    - python
```
Comment (baileythegreen) [inline glossary.yml:4756]:
```suggestion
    - python
    - pip
```

File: glossary.yml [hunk 4]
```diff
@@ -4481,6 +4481,17 @@
       Phony targets are often used to store commonly-used commands in a [Makefile](#makefile).
 
 
+- slug: pip
+  ref:
+    - python
```
Comment (baileythegreen) [inline glossary.yml:4486]:
```suggestion
    - pypi
    - python
```

**Review by baileythegreen — CHANGES_REQUESTED** [review]
I've suggested some additional crosslinks between the two entries (I *think* these should compile okay, since both are being added simultaneously), and changes to line breaks.

**Review by callumrollo — COMMENTED** [review]
Incorporating changes from @baileythegreen. Thanks for the improvements! Hopefully the simultaneous cross links will work

**Review by baileythegreen — APPROVED** [review]

**Comment by elletjies** [comment]
@all-contributors please add @callumrollo for translation

**Pull request merged**